examples/testdata/cat.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+// main is the same as cat.
+func main() {
+	for _, file := range os.Args {
+		bytes, err := os.ReadFile(file)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "couldn't read %s: %v", file, err)
```

Comment:
IMHO: generally we should do something like this instead of panic because panics from tinygo are infinitely difficult to troubleshoot.

Reply:
switched to just error code which is just as well as panic when there is only one function (_start). Removed the printf because it added 140KB to the wasm